recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -95,41 +84,44 @@ def build_requirements(self):
 
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, 17 if Version(self.version) >= "2.9.1" else 14)
+            check_min_cppstd(self, 17)
 
         minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
         if not minimum_version:
-            self.output.warn(f"{self.name} requires C++14. Your compiler is unknown. Assuming it supports C++14.")
+            self.output.warn(f"{self.name} requires C++17. Your compiler is unknown. Assuming it supports C++17.")
         elif Version(self.settings.compiler.version) < minimum_version:
-            raise ConanInvalidConfiguration(f"{self.name} requires C++14, which your compiler does not support.")
+            raise ConanInvalidConfiguration(f"{self.name} requires C++17, which your compiler does not support.")
 
     def source(self):
-        files.get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
     def build(self):
-        files.apply_conandata_patches(self)
-        cmake = self._configure_cmake()
+        apply_conandata_patches(self)
+        cmake = CMake(self)
+        cmake.configure(build_script_folder=join("tensorflow", "lite"))
         cmake.build()
 
-    @functools.lru_cache(1)
-    def _configure_cmake(self):
-        cmake = CMake(self)
-        cmake.definitions.update({
+    def generate(self):
```

Comment:
This should be just after requirements along with https://github.com/conan-io/conan-center-index/pull/12686/files#diff-834643045c143a557b3010cae32525f45e623ccb4c2a6ba041a056757ec6aefbR88-R92

Reply:
The work around is in my PR for https://github.com/conan-io/conan-center-index/pull/12390#issuecomment-1245755734

Reply:
I've applied it and it does the trick. Thanks! Did you find out why this happens and only happens on windows?

Reply:
Case insensitive 😒 

There a BUILD file that doesn't collide else where

Reply:
Oh, I see! Thanks for the info :)